### PR TITLE
feat(lang): Add Bengali (bn) translations

### PR DESCRIPTION
**Pull request by themaruf**
## Description
Add completed translations for Bengali language.

## Specific Changes proposed
New file of Bengali (bn.json) language strings.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [x] If necessary, more likely in a feature request than a bug fix
  - [x] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: lang/bn.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "Duration": "ব্যাপ্তিকাল",
+    "Remaining Time": "অবশিষ্ট সময়",
+    "Stream Type": "স্ট্রিমের ধরন",
+    "LIVE": "লাইভ",
+    "Seek to live, currently behind live": "লাইভ দেখুন, বর্তমানে লাইভের পিছনে আছে",
+    "Seek to live, currently playing live": "লাইভ দেখুন, বর্তমানে লাইভে আছে",
+    "Loaded": "লোড হয়েছে",
+    "Progress": "প্রোগ্রেস",
+    "Progress Bar": "প্রোগ্রেস বার",
+    "progress bar timing: currentTime={1} duration={2}": "{২} এর {১}",
```
Comment (mister-ben) [inline lang/bn.json:17]: These tokens need to remains as exactly `{1}` and `{2}` for the replacement to work.

File: lang/bn.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    "Casual": "ক্যাজুয়াল",
+    "Script": "স্ক্রিপ্ট",
+    "Small Caps": "ছোট ক্যাপস",
+    "Reset": "রিসেট",
+    "restore all settings to the default values": "সমস্ত সেটিংস ডিফল্ট মানগুলিতে পুনরুদ্ধার করুন",
+    "Done": "সম্পন্ন",
+    "Caption Settings Dialog": "ক্যাপশন সেটিংস ডায়ালগ",
+    "Beginning of dialog window. Escape will cancel and close the window.": "ডায়লগ উইন্ডোর শুরু। Esc কী বাতিল করবে এবং উইন্ডো বন্ধ করবে।",
+    "End of dialog window.": "ডায়ালগ উইন্ডোর শেষ।",
+    "{1} is loading.": "{১} লোড হচ্ছে.",
```
Comment (mister-ben) [inline lang/bn.json:86]:
```suggestion
    "{1} is loading.": "{1} লোড হচ্ছে.",
```

**Review by mister-ben — CHANGES_REQUESTED** [review]
See comments about the replacement tokens, otherise looks good.

File: lang/bn.json (hunk 1 above)
Comment (mister-ben) [reply lang/bn.json:17]: Like this:

```suggestion
    "progress bar timing: currentTime={1} duration={2}": "{2} এর {1}",
```

**Review by misteroneill — APPROVED** [review]
LGTM, thank you for the PR!

**Review by mister-ben — APPROVED** [review]

**Pull request merged**